modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/computation/BulkScrollerComputation.java

```diff
@@ -284,20 +287,21 @@ protected void updateStatusAsScrolling(ComputationContext context, String comman
                 BulkCodecs.getStatusCodec().encode(delta));
     }
 
-    protected void updateStatusAfterScroll(ComputationContext context, String commandId, String errorMessage) {
-        updateStatusAfterScroll(context, commandId, 0, errorMessage, false);
+    protected void updateStatusAfterScroll(ComputationContext context, String commandId, String errorMessage,
+            Integer errorCode) {
+        updateStatusAfterScroll(context, commandId, 0, errorMessage, errorCode, false);
     }
 
     protected void updateStatusAfterScroll(ComputationContext context, String commandId, long documentCount,
             boolean limited) {
-        updateStatusAfterScroll(context, commandId, documentCount, null, limited);
+        updateStatusAfterScroll(context, commandId, documentCount, null, 0, limited);
```

Comment:
```suggestion
        updateStatusAfterScroll(context, commandId, documentCount, null, null, limited);
```

Reply:
yep, thanks Kevin. I completely forgot it.

Reply:
No problem 😉